components/scream/data/scream_input.yaml

```diff
@@ -49,12 +49,14 @@ Atmosphere Driver:
 
 SCREAM:
 
-  Start Date: 01-01-2000 # For CIME cases, do NOT set this! It will be set by CIME's RUN_STARTDATE
+  Start Date: "${RUN_STARTDATE}"
 
   Input Files:
-    input_root                  : default # this will default to DIN_LOC_ROOT for the current machine
+    input_root                  : "${DIN_LOC_ROOT}"
 
 
+  Test Var: "<${ATM_GRID} : ne4np4 => 42 : 420>"
```

Comment:
Might add a second example `Test Var 2: ...` where you show how to use 2+ matching values for the cond var. So it's clearer that this is not (just) a ternary op.

Reply:
@bartgol , this `Test Var` item should not have been left in this file to begin with. I temporarily inserted it just to test that switch statement syntax worked. I added some comments to the top of buildnml to hopefully clarify how the user can use switch statements.